pkg/openid4ci/errors.go

```diff
@@ -0,0 +1,23 @@
+package openid4ci
+
+const (
+	module = "OCI"
+
+	NoClientConfigProvidedCode  = module + "0-0001"
+	NoClientConfigProvidedError = "NO_CLIENT_CONFIG_PROVIDED"
+
+	ClientConfigNoUserDidProvidedCode  = module + "0-0002"
+	ClientConfigNoUserDidProvidedError = "CLIENT_CONFIG_NO_USER_DID_PROVIDED"
+
+	ClientConfigNoClientIDProvidedCode  = module + "0-0003"
+	ClientConfigNoClientIDProvidedError = "CLIENT_CONFIG_NO_CLIENT_ID_PROVIDED"
+
+	ClientConfigNoSignerProviderProvidedCode  = module + "0-0004"
+	ClientConfigNoSignerProviderProvidedError = "CLIENT_CONFIG_NO_SIGNER_PROVIDER_PROVIDED"
+
+	ClientConfigNoDIDResolverProvidedCode  = module + "0-0005"
+	ClientConfigNoDIDResolverProvidedError = "CLIENT_CONFIG_DID_RESOLVER_PROVIDED"
+
+	PreAuthorizedCodeRequiredCode  = module + "0-0001"
+	PreAuthorizedCodeRequiredError = "PRE_AUTHORIZED_CODE_REQUIRED"
```

Comment:
Why cant we use `iota`, and let `walleterror` build the whole error code based on scenario.

Example: https://github.com/hyperledger/aries-framework-go/blob/main/pkg/controller/command/vcwallet/command.go#L269